pyincore_viz/geoutil.py

```diff
@@ -371,11 +371,15 @@ def get_gdf_map(datasets: list):
         # (min_lat, min_lon, max_lat, max_lon)
         bbox_all = [9999, 9999, -9999, -9999]
 
-        for dataset in datasets:
-            # maybe this part should be moved to Dataset Class
-            gdf = gpd.read_file(dataset.local_file_path)
-            geo_data = ipylft.GeoData(
-                geo_dataframe=gdf, name=dataset.metadata['title'])
+        for i, dataset in enumerate(datasets):
+            if isinstance(dataset, Dataset):
+                gdf = gpd.read_file(dataset.local_file_path)
```

Comment:
There is a method in pyincore Dataset that does that. Could we use that method?

```
 def get_dataframe_from_shapefile(self):
        """Utility method for reading different standard file formats: GeoDataFrame from shapefile.

        Returns:
            obj: Geopanda's GeoDataFrame.

        """
        # read shapefile directly by Geopandas.read_file()
        # It will preserve CRS information also
        gdf = gpd.read_file(self.local_file_path)

        return gdf
 ```

Reply:
oh, nice, updated. Thanks